spec/unit/gateway/contentful_gateway_spec.rb

```diff
@@ -178,4 +178,38 @@
       ])
     end
   end
+
+  context '#When no case study is found' do
```

Comment:
```suggestion
  context '#get_case_study (none found example)' do
```

Maybe rename to something like that? So it aligns with the naming of the other contexts.

Reply:
Makes sense changed in 
https://github.com/DFE-Digital/single-place-for-education-frontend/pull/9/commits/f6a135520d6e96f640b524d4e065244a29a3cc3c